in_toto/runlib.go

```diff
@@ -80,6 +89,28 @@ func RecordArtifacts(paths []string) (map[string]interface{}, error) {
 				if info.IsDir() {
 					return nil
 				}
+
+				// check for symlink and evaluate the last element in a symlink chain
+				// via filepath.EvalSymlinks. We use filepath.EvalSymlinks here, because
+				// filepath.EvalSymlinks returns last element in a symlink chain.
+				// With os.Readlink() we would just read the next element in a possible symlink chain.
+				if info.Mode()&os.ModeSymlink == os.ModeSymlink {
+					evalSym, err := filepath.EvalSymlinks(path)
+					if err != nil {
+						return err
+					}
+					// We recursively call RecordArtifacts() to follow the new path
+					// Note: filepath.EvalSymlinks() handles symlink chains and throws an error if
+					// it detects such a chain like: chain1 -> chain2, chain2 -> chain1
```

Comment:
Is this comment still true? How does this it relate to your discovery in https://github.com/in-toto/in-toto-golang/pull/55#issuecomment-641152088?

Reply:
This will get updated, too

Reply:
This should be fixed.. I leave resolving the conversation to you, just in case you found something else or I misunderstood you.